src/electron-main.js

```diff
@@ -35,7 +35,6 @@ const tray = require('./tray');
 const vectorMenu = require('./vectormenu');
 const webContentsHandler = require('./webcontents-handler');
 const updater = require('./updater');
-const { migrateFromOldOrigin } = require('./originMigrator');
```

Comment:
Should this module also be removed?

Reply:
I just didn't copy it over to the new repo